src/main/java/com/slisenko/education/html/PageDownloader.java

```diff
@@ -40,4 +45,49 @@ public void download(String link, String path) throws DownloadException {
             throw new DownloadException("Page loading error from link " + link + " to file " + path, e);
         }
     }
+
+    /**
+     * Reads the file to a string
+     * @param path File path
+     * @return A string with the contents of the file
+     * @throws DownloadException
+     */
+    public String fileToString(String path) throws DownloadException {
```

Comment:
conceptually looks strange when fileToString method throws DownloadException, as it doesn't do any download
May be throw different type of exception